docker-compose.yml

```diff
@@ -1,14 +1,14 @@
-version: '3.0'
 services:
-  leon-ai:
-    container_name: 'leon-ai'
+  leon:
+    container_name: 'leon'
     build:
       context: './'
-    image: 'leonai/leon'
```

Comment:
As the organization is called `leon-ai` everywhere, not `leonai`, I find it nicer to call the image `leon-ai`.
This change is not strictly needed for this fix, but I guess it is a nice change to do, feel free to disagree, I can revert that one.